### PR TITLE
Allow everyone to pray

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This put the verb on /client/ instead of /mob/

fixes https://github.com/tgstation/TerraGov-Marine-Corps/issues/7571

image

## Changelog
:cl:
qol: Allow everyone to pray!
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
fixes https://github.com/tgstation/TerraGov-Marine-Corps/issues/7571

**Comment by lbnesquik** [comment]
image

File: code/modules/admin/verbs/pray.dm [hunk 1]
```diff
@@ -11,14 +11,18 @@
...
 	var/liaison = FALSE
 
-	if(iscorporateliaisonjob(job))
-		liaison = TRUE
+	if(ismob(usr))
+		if(ishuman(usr))
+			for(var/i in GLOB.human_mob_list)
+				var/mob/living/carbon/human/H = i
+				if(iscorporateliaisonjob(H.job))
+					liaison = TRUE
```
Comment (BraveMole) [inline code/modules/admin/verbs/pray.dm:25]: what the fuck is that ?????

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by Losenis** [comment]
image

File: code/modules/admin/verbs/pray.dm [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
+	if(handle_spam_prevention(msg, MUTE_PRAY))
 		return
 
 	var/mentor_msg = msg
 	var/liaison = FALSE
 
-	if(iscorporateliaisonjob(job))
-		liaison = TRUE
+	if(ismob(usr))
+		if(ishuman(usr))
```
Comment (BraveMole) [inline code/modules/admin/verbs/pray.dm:21]: a human is already a mob

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/pray.dm (hunk 2 above)
Comment (lbnesquik) [reply code/modules/admin/verbs/pray.dm:21]: agh

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**